core/src/main/java/co/aikar/commands/annotation/Flags.java

```diff
@@ -26,6 +26,9 @@
 import java.lang.annotation.Retention;
 import java.lang.annotation.RetentionPolicy;
 
+/**
+ * Sets the flags each parameter uses in this method.
```

Comment:
Flags are used to configure and change behavior of context resolution. For adding restrictions on a resolved context, see Conditions